pkg/resource/logging-config/alloy/logging.alloy.template

```diff
@@ -1,11 +1,11 @@
 // Kubernetes pods logs
 {{- if .SupportPodLogs }}
 loki.source.podlogs "kubernetes_pods" {
-  forward_to      = [loki.relabel.kubernetes_pods.receiver]
```

Comment:
It's funny how some indentation changed in multiple places.
Is it a change with `gofmt` behaviour? Depending on `go` version?
We should make sure this is legit and a one-time change.

Reply:
No it's not, the issue is we were using both tabs and spaces in the same template

Reply:
I just made sure we use the same everywhere on this file

Reply:
Oh, it's not a go file, that's why!
`gofmt` does not run on this.